### PR TITLE
Add new operators

### PR DESCRIPTION
**Pull request by Mr-Martinez**
**Fixes**
#164 
#152 
#148 

**Solution**
Add new operator files and appropriate changes

### CODE REVIEW

File: src/main/create-operator-json/operators/vigil.js [hunk 1]
```diff
@@ -11,5 +11,6 @@ vigil.addCounterNode(operatorId.dokkaebi, counterType.soft,  "Vigil's ERC-7 allo
 vigil.addCounterNode(operatorId.blackbeard, counterType.minor, "Vigil's BOSG.12.2 shotgun can destroy Blackbeard's Rifle-Shield in a single shot.");
 vigil.addCounterNode(operatorId.twitch, counterType.minor,  "Vigil's ERC-7 allows him to remove himself from Twitch's Drone camera.");
 vigil.addCounterNode(operatorId.lion, counterType.hard, "Vigil cannot be detected by Lion's EE-ONE-D drone scan when his ER7-C gadget is active.");
+vigil.addCounterNode(operatorId.iana, counterType.minor, "Vigils Video Disruptor will make him unseen by Ianas Hologram");
```
Comment (BurkusCat) [inline src/main/create-operator-json/operators/vigil.js]: Vigil's ERC-7 makes him invisible to Iana's Hologram

File: src/main/create-operator-json/operators/maestro.js [hunk 2]
```diff
@@ -7,4 +7,6 @@ import r6operators from "r6operators";
 
 let maestro = new Operator(r6operators.maestro, operatorId.maestro, "Operation Para Bellum");
 
+maestro.addCounterNode(operatorId.iana, counterType.soft, "Maestro's Evil Eyes can destroy the Hologram with its laser, and can detect its lack of heat signature");
```
Comment (BurkusCat) [inline src/main/create-operator-json/operators/maestro.js]: ...can destroy **Iana's** Hologram with its laser...

File: src/main/create-operator-json/operators/jager.js [hunk 3]
```diff
@@ -14,6 +14,6 @@ jager.addCounterNode(operatorId.nomad, counterType.soft, "Jäger's ADS can destr
 jager.addCounterNode(operatorId.glaz, counterType.minor,  "Jäger's ADS can intercept smoke grenades. Smoke grenades are a key part of Glaz's kit.");
 jager.addCounterNode(operatorId.ash, counterType.minor, "Jäger's ADS can intercept Ash's Breaching Rounds.");
 jager.addCounterNode(operatorId.fuze, counterType.soft, "Jäger's ADS can intercept Fuze's Cluster Charges. Fuze gets three charges each with five sub-grenades (15 total) and Jäger's ADS can block two projectiles each (6 total). This means ADS is only partially effective against a Fuze attack.");
-
+jager.addCounterNode(operatorId.kali, counterType.hard, "Jäger's ADS can Kali grenades.");
```
Comment (BurkusCat) [inline src/main/create-operator-json/operators/jager.js]: Jäger's ADS can intercept Kali's Explosive Lances.

File: src/main/create-operator-json/operators/kali.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+import counterType from '../core/counterTypeEnums';
+import operatorId from '../core/operatorIdEnum';
+import Operator from '../core/operatorFactory';
+import r6operators from "r6operators";
+
+let kali = new Operator(r6operators.kali, operatorId.kali, "Operation Shifting Tides");
+
+kali.addCounterNode(operatorId.goyo, counterType.hard, 'Kali can destroy Goyos shields from the other side.');
```
Comment (BurkusCat) [inline src/main/create-operator-json/operators/kali.js]: A few style things, would you be able to do apply these to all of the relations you are adding, please?

- Use the fuller name for each gadget e.g. Kali's CSRX 300 Rifle, Kali's Explosive Lances, Goyo's Volcán Shield etc.
- Apostrophes for each operator. e.g. Goyo's

File: src/main/create-operator-json/operators/kali.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+kali.addCounterNode(operatorId.bandit, counterType.hard, 'Kali can destroy Bandits batteries.');
+kali.addCounterNode(operatorId.mute, counterType.hard, 'Kali can destroy Mutes jammers.');
+kali.addCounterNode(operatorId.maestro, counterType.hard, 'Kali can destroy Maestros camera.');
+kali.addCounterNode(operatorId.castle, counterType.hard, 'Kali can Castle doors and windows.');
```
Comment (BurkusCat) [inline src/main/create-operator-json/operators/kali.js]: Kali's Explosive Lances can destroy Castle's Armor Panels.

File: src/main/create-operator-json/operators/kali.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+kali.addCounterNode(operatorId.clash, counterType.hard, 'Kali can stagger Clash Shield, Clash needs coordination to bring down');
```
Comment (BurkusCat) [inline src/main/create-operator-json/operators/kali.js]: I would potentially reword this one. "Kali's CSRX 300 Rifle shots can stagger Clash and knock her Shield aside."

File: src/main/create-operator-json/operators/wamai.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+import counterType from '../core/counterTypeEnums';
+import operatorId from '../core/operatorIdEnum';
+import Operator from '../core/operatorFactory';
+import r6operators from "r6operators";
+
+let wamai = new Operator(r6operators.wamai, operatorId.wamai, "Operation Shifting Tides");
+
+wamai.addCounterNode(operatorId.thatcher, counterType.hard,  "Wamai pulls Thatcer EMPs");
```
Comment (BurkusCat) [inline src/main/create-operator-json/operators/wamai.js]: Thatcher

File: src/main/create-operator-json/operators/wamai.js [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+wamai.addCounterNode(operatorId.ash, counterType.hard,  "Wamai pulls Ashs Breacing Charge");
```
Comment (BurkusCat) [inline src/main/create-operator-json/operators/wamai.js]: Breaching

File: src/main/create-operator-json/operators/wamai.js [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+wamai.addCounterNode(operatorId.fuze, counterType.hard,  "Wamai pulls Fuzes Cluster Charges");
+wamai.addCounterNode(operatorId.capitao, counterType.hard,  "Wamai pulls Capitaos Smoke and Fire Bolts");
```
Comment (BurkusCat) [inline src/main/create-operator-json/operators/wamai.js]: Capitão's

File: src/main/create-operator-json/operators/wamai.js [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+wamai.addCounterNode(operatorId.ying, counterType.hard,  "Wamai pulls Yings Cadelas");
+wamai.addCounterNode(operatorId.nomad, counterType.hard,  "Wamai pulls Nomads Airjabs");
```
Comment (BurkusCat) [inline src/main/create-operator-json/operators/wamai.js]: Full stops/periods are needed at the end of each counter.

File: src/main/create-operator-json/operators/wamai.js [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+wamai.addCounterNode(operatorId.zofia, counterType.hard,  "Wamai pulls Zofias Concussion and Impact Grenades");
+wamai.addCounterNode(operatorId.gridlock, counterType.minor,  "Wamai pulls Gridlocks Trax Stinger but will not destroy");
+wamai.addCounterNode(operatorId.kali, counterType.hard,  "Wamai pulls Kalis LV Lances");
+wamai.addCounterNode(operatorId.recruit, counterType.hard,  "Wamai pulls frags, smokes, and stuns");
```
Comment (BurkusCat) [inline src/main/create-operator-json/operators/wamai.js]: I potentially would avoid having a recruit counter. We could possibly remove the smoke -> recruit counter as well.

File: src/main/create-operator-json/operators/vigil.js (hunk 1 above)
Comment (Mr-Martinez) [reply src/main/create-operator-json/operators/vigil.js]: fixed

File: src/main/create-operator-json/operators/maestro.js (hunk 2 above)
Comment (Mr-Martinez) [reply src/main/create-operator-json/operators/maestro.js]: fixed

File: src/main/create-operator-json/operators/gridlock.js [hunk 12]
```diff
@@ -9,6 +9,6 @@ let gridlock = new Operator(r6operators.gridlock, operatorId.gridlock, "N/A");
 
 gridlock.addCounterNode(operatorId.vigil, counterType.soft, "Gridlock's Trax Stingers can impede Vigil's strong roaming capabilities.");
 gridlock.addCounterNode(operatorId.caviera, counterType.soft, "Gridlock's Trax Stingers can impede Caveira's strong roaming capabilities.");
-gridlock.addCounterNode(operatorId.oryx, counterType.soft, "Gridlock's Trax Stingers can be used to knock Oryx to the ground when dashing.");
+gridlock.addCounterNode(operatorId.oryx, counterType.soft, "Gridlock's Trax Stingers interrupt Oryx's Remah Daashes.");
```
Comment (BurkusCat) [inline src/main/create-operator-json/operators/gridlock.js]: Dashes

File: src/main/create-operator-json/operators/jager.js (hunk 3 above)
Comment (Mr-Martinez) [reply src/main/create-operator-json/operators/jager.js]: fixed

File: src/main/create-operator-json/operators/kali.js (hunk 4 above)
Comment (Mr-Martinez) [reply src/main/create-operator-json/operators/kali.js]: fixed

File: src/main/create-operator-json/operators/wamai.js (hunk 7 above)
Comment (Mr-Martinez) [reply src/main/create-operator-json/operators/wamai.js]: fixed

File: src/main/create-operator-json/operators/kali.js (hunk 5 above)
Comment (Mr-Martinez) [reply src/main/create-operator-json/operators/kali.js]: fixed

File: src/main/create-operator-json/operators/wamai.js (hunk 8 above)
Comment (Mr-Martinez) [reply src/main/create-operator-json/operators/wamai.js]: fixed

File: src/main/create-operator-json/operators/wamai.js (hunk 9 above)
Comment (Mr-Martinez) [reply src/main/create-operator-json/operators/wamai.js]: fixed

File: src/main/create-operator-json/operators/wamai.js (hunk 10 above)
Comment (Mr-Martinez) [reply src/main/create-operator-json/operators/wamai.js]: fixed

File: src/main/create-operator-json/operators/wamai.js [hunk 13]
```diff
@@ -7,15 +7,14 @@ import r6operators from "r6operators";
...
-wamai.addCounterNode(operatorId.kali, counterType.hard,  "Wamai pulls Kalis LV Lances");
-wamai.addCounterNode(operatorId.recruit, counterType.hard,  "Wamai pulls frags, smokes, and stuns");
+wamai.addCounterNode(operatorId.thatcher, counterType.hard,  "Wamai's MAG-NET System pulls Thathcer EMP Grenades.");
+wamai.addCounterNode(operatorId.ash, counterType.hard,  "Wamai's MAG-NET System pulls Ash's Breaching Charge.");
+wamai.addCounterNode(operatorId.fuze, counterType.hard,  "Wamai's MAG-NET System pulls Fuze's Cluster Charges.");
+wamai.addCounterNode(operatorId.capitao, counterType.hard,  "Wamai's MAG-NET System pulls Capitão's Smoke and Fire Bolts.");
+wamai.addCounterNode(operatorId.ying, counterType.hard,  "Wamai's MAG-NET System pulls Ying's Cadelas.");
+wamai.addCounterNode(operatorId.nomad, counterType.hard,  "Wamai's MAG-NET System pulls Nomad's Airjabs.");
+wamai.addCounterNode(operatorId.zofia, counterType.hard,  "Wamai's MAG-NET System pulls Zofia's Concussion and Impact Grenades.");
+wamai.addCounterNode(operatorId.gridlock, counterType.minor,  "Wamai's MAG-NET System pulls Gridlock's Trax Stinger but will not destroy.");
```
Comment (BurkusCat) [inline src/main/create-operator-json/operators/wamai.js]: ...but does not destroy it.

File: src/main/create-operator-json/operators/wamai.js (hunk 11 above)
Comment (Mr-Martinez) [reply src/main/create-operator-json/operators/wamai.js]: fixed

File: src/main/create-operator-json/operators/mute.js [hunk 14]
```diff
@@ -12,8 +12,9 @@ mute.addCounterNode(operatorId.thermite, counterType.hard, "Mute's Signal Disrup
...
 mute.addCounterNode(operatorId.twitch, counterType.hard, "If placed correctly, Mute's Signal Disruptor can disrupt Twitch's Drone.");
 mute.addCounterNode(operatorId.dokkaebi, counterType.hard, "When standing within range of Mute's Signal Disruptor, a defender's phone will not ring when Dokkaebi activates her Logic Bomb. A ringing phone will stop if you step within range of a Signal Disruptor.");
-
 mute.addCounterNode(operatorId.jackal, counterType.soft, "Mute's Signal Disruptor covers Jackal's Eyenox in static. This makes vision difficult and prevents Jackal from scanning Defender footprints.")
 mute.addCounterNode(operatorId.lion, counterType.soft, "When standing within range of Mute's Signal Disruptor, a defender can freely move without being detected by Lion's EE-ONE-D drone.");
 mute.addCounterNode(operatorId.blitz, counterType.soft, "When standing within range of Mute's Signal Disruptor, Blitz's Flash Shield will be disrupted.");
+mute.addCounterNode(operatorId.iana, counterType.hard, "Mute's Signal Disruptor will neutralize Iana's hologram");
```
Comment (BurkusCat) [inline src/main/create-operator-json/operators/mute.js]: Full stop/period needed at end

File: src/main/create-operator-json/operators/kali.js (hunk 6 above)
Comment (Mr-Martinez) [reply src/main/create-operator-json/operators/kali.js]: fixed

**Comment by Mr-Martinez** [comment]
Thanks for the grammar catches!

**Pull request merged**